infrastructure/modules/armonik/core-configmap.tf

```diff
@@ -5,21 +5,21 @@ resource "kubernetes_config_map" "core_config" {
     namespace = var.namespace
   }
   data = merge(var.extra_conf.core, {
-    Components__TableStorage             = local.table_storage_adapter
-    Components__ObjectStorage            = local.object_storage_adapter
-    Components__QueueStorage             = local.queue_storage_adapter
```

Comment:
I would keep this one to make the infrastructure compatible with both previous and current versions of core.

Reply:
Done